opennms/scripts/onms-post-init.sh

```diff
@@ -34,16 +34,18 @@ command -v jq >/dev/null 2>&1   || { echo >&2 "jq is required but it's not insta
 # Wait for dependencies
 wait_for ${OPENNMS_SERVER}:8980
 
+ONMS_AUTH="admin:${OPENNMS_ADMIN_PASS}"
+
 if [[ ${ENABLE_GRAFANA} == "true" ]]; then
   # Configure Grafana Endpoint for Reports
-  ID=$(curl -u admin:admin http://${OPENNMS_SERVER}:8980/opennms/rest/endpoints/grafana 2>/dev/null | jq ".[] | select(.uid=\"$(hostname)\") | .id") || true
+  ID=$(curl -sSf -u "${ONMS_AUTH}" http://${OPENNMS_SERVER}:8980/opennms/rest/endpoints/grafana | jq ".[] | select(.uid=\"$(hostname)\") | .id") || true
   if [[ $ID == "" ]]; then
-    GRAFANA_KEY=$(curl -u "admin:${GF_SECURITY_ADMIN_PASSWORD}" -X POST -H "Content-Type: application/json" -d "{\"name\":\"$(hostname)\",\"role\": \"Viewer\"}" "http://${GRAFANA_SERVER}:3000/api/auth/keys" 2>/dev/null | jq .key - | sed 's/"//g')
+    GRAFANA_KEY=$(curl -sSf -u "${ONMS_AUTH}" -X POST -H "Content-Type: application/json" -d "{\"name\":\"$(hostname)\",\"role\": \"Viewer\"}" "http://${GRAFANA_SERVER}:3000/api/auth/keys" | jq -r .key)
```

Comment:
Why use the OpenNMS password for the Grafana ReST API?

Reply:
Because I made a mistake and my testing didn’t catch it. ;-) Good eye! Thanks!